### DISCUSSION TITLE
Beam Time

### DISCUSSION THREAD

**Discussion by DoomLordNV** [arkypita/LaserGRBL#2104] [Q&A]
Hi there. Is there anyway to track beam time in Laser GRBL?

**Comment by arkypita** [comment]
It's something I've been thinking about for some time, perhaps with the possibility of sending a report when the laser "dies" which could end up in a global knowledge of the average life of the lasers of the various manufacturers.

Unfortunately it is not so easy to determine. The use of the laser in terms of hours of life strictly depends on the power at which it is turned on and, in some works such as the gray scales, this is dynamic.

A time/power factor should therefore be normalized.

Also there are people who own more than one laser machine, so one would have to find ways to count separate values for different machines, but there's no way for me to recognize them.

So at the moment I have avoided putting this information, but I'm thinking of doing it (at least for those who have only one laser)

**Reply by arkypita** [reply]
PS. I am working on it :-)

**Comment by arkypita** [comment]
@DoomLordNV What do you think about this version?
https://github.com/arkypita/LaserGRBL/releases/tag/v5.1.0

**Comment by DoomLordNV** [comment]
Nice! I think I might be able to put this to good use for sure!

On Sun, Aug 6, 2023 at 7:24 AM arkypita ***@***.***> wrote:

> @DoomLordNV <https://github.com/DoomLordNV> What do you think about this
> version?
> https://github.com/arkypita/LaserGRBL/releases/tag/v5.1.0
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/arkypita/LaserGRBL/discussions/2104#discussioncomment-6651108>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBMFODEJTOBEBHALF6C27ELXT6SJLANCNFSM6AAAAAA2QJH6DY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 


*Arlis Kelly **| Production Manager (NV)*

Reno Operations | 4835 Longley Ln | Reno, NV 89502

e: ***@***.***| m *+1.775.790.9097*
